game.py

```diff
@@ -20,8 +20,35 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
-
+    # build a dict to store letter and status (T or F) for each letter in word
+    letter_statuses_dict = build_letter_status_dict(snowman_word)
+    # define a wrong guesses list
+    wrong_guesses_list = []
+
+    # loop condition: wrong guesses < MAX wrong guesses
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        # get user input
+        user_input = get_letter_from_user(letter_statuses_dict, wrong_guesses_list)
+        # check if user_input is in the word
+        # if yes, change status of the letter in the letter_statuses_dict
+        if user_input in letter_statuses_dict:
+            letter_statuses_dict[user_input] = True
+            # check is_word_guessed, if yes, player wins, print message, exit;
+            if is_word_guessed(snowman_word, letter_statuses_dict):
+                print("Congratulations, you win!")   
+                return
+        # else: add to wrong guesses list;
+        else: 
+            wrong_guesses_list.append(user_input)
+        # print_word_progress_string
+        print_word_progress_string(snowman_word, letter_statuses_dict)
+        # print list of wrong guesses
+        print(f"Wrong guesses: {wrong_guesses_list}")
+        # print snowman graphic
+        print_snowman_graphic(len(wrong_guesses_list))
+        
+    # player loses, print message
+    print(f"Sorry, you lose! The word was {snowman_word}")
```

Comment:
You've made a great choice here as well by recognizing that the win condition is handled in the loop so we don't have to worry with any conditionals outside of the loop! Great catch!